CIP-1694/README.md

```diff
@@ -753,8 +754,6 @@ Client applications should, however, perform the usual sanity checks when fetchi
 DRep retirement certificates include:
 
 * a DRep ID
-* the epoch number after which the DRep will retire
-* an optional anchor
```

Comment:
Why remove the anchor for retirements?

Reply:
I could add that back, it's just not in the spec and I didn't see a use-case that isn't covered by the regular transaction metadata.